_posts/2016-05-16-here-comes-the-telemetry.md

```diff
@@ -0,0 +1,159 @@
+
+First, add the new dependency with composer:
+
+```
+$ composer require --dev atoum/reports-extension
+```
+
+THen enable the report in your configuration file, `.atoum.php`:
+
+```
+<?php
+
+use mageekguy\atoum\reports\telemetry;
+use mageekguy\atoum\writers\std;
+
+$script->addDefaultReport();
+
+$telemetry = new telemetry();
+$telemetry->addWriter(new std\out());
+$runner->addReport($telemetry);
+```
+
+Now each time you run tour test suite, results will be sent to the telemetry.
+
+Because the dashboards are publicly accessible, by default everything will be sent anonymously. If you want to let us 
+know who you are and which project is producing those data, you can add the following lines:
+
+```
+<?php
+
+$telemetry->readProjectNameFromComposerJson(__DIR__ . '/composer.json');
+
+// Or
+
+$telemetry->setProjectName('my/project');
+
+// And optionally
+
+$telemetry->sendAnonymousProjectName();
+```
+
+## What do we collect?
+
+Now that you know how to enable the telemetry report, you'll probably want to know what is collected from your test 
+suites. atoum will compute some numbers about your test suites, here is the full list:
+
+* Number of test classes,
+* Number of test methods,
+* Number of test methods by status (void, uncomplete, failed, ...),
+* Number of assertions,
+* Number of assertions by status (faile and passed),
+* Number of exceptions,
+* Number of errors,
+* Lines coverage value,
+* Branches coverage value,
+* Paths coverage value,
+* Test suite duration,
+* Memory consumption.
+
+It will also tag them with informations about the execution environment:
+
+* PHP version,
+* atoum version,
+* Operating system,
+* Platform architecture,
+* CI environment name,
+* Vendor name
+* Project name
+
+Of course, if you chose to send anonymous data, vendor and project names will be anonymized.
+
+The data we collect will be kept as long as possible. They will eventually be down-sampled each month to keep the 
+database as small as possible without loosing anything.
+
+## Numbers, numbers, numbers!
+
+Now you might ask what is the point of this platform. This is simple: we want to be able to measure things and improve 
+them based on real feedbacks. Moreover, this feedback will be produced at no cost for users.
+
+> If you can't measure it, you can't improve it.
+>
+> -- [Peter Drucker](https://en.wikipedia.org/wiki/Peter_Drucker)
+
+atoum has been written with TDD in mind. We often think TDD is limited to design and code. But one of its key concept 
+is the feedback. Without this feedback you are left alone to make decision. Measuring things with the telemtry will let 
+us make decision with real numbers. For example, if we want to deprecate a PHP version from the list of supported ones 
+we'll only have to look at the telemetry and see the usage breakdown to measure the impact of such a change.
+
+We'll also be able to monitor performances across releases of atoum: the telemetry will be notified by github each time 
```

Comment:
if the reports are tagged with atoum's version. Wouldn't it be better to use this to monitor performance ? 
A better example of this github notification feature wouldn't be to understand in which time a new release is used ?